app/utilities/messages.py

```diff
@@ -21,3 +21,15 @@
     status.HTTP_200_OK: {"description": "User recovered"},
     status.HTTP_404_NOT_FOUND: {"description": "User not found"},
 }
+
+POST_PUBLIC_KEY_RESPONSES = {
+    status.HTTP_201_CREATED: {"description": "User Public key saved"}
+}
+
+CREATE_TOKEN_RESPONSES = {status.HTTP_201_CREATED: {"description": "Token created"}}
```

Comment:
Para mantener la convención:
```suggestion
POST_TOKEN_RESPONSES = {status.HTTP_201_CREATED: {"description": "Token created"}}
```

Reply:
Echo